bindings/c/test/mako/mako.cpp

```diff
@@ -1053,6 +1047,7 @@ int parseArguments(int argc, char* argv[], Arguments& args) {
 			{ "disable_ryw", no_argument, NULL, ARG_DISABLE_RYW },
 			{ "json_report", optional_argument, NULL, ARG_JSON_REPORT },
 			{ "bg_file_path", required_argument, NULL, ARG_BG_FILE_PATH },
+			{ "export", optional_argument, NULL, ARG_EXPORT },
```

Comment:
How about `export_sketch`? I put it as a flag with an optional argument so that it will create a file with a default name, say `sketch_data.json` unless the user would manually specify a file.

Reply:
`export_sketch_path` would work